### PR TITLE
small cleanup refactoring

### PR DESCRIPTION
**Pull request by pepoviola**
- fix small bugs related to collators groups.
- set default concurrency to 4
- Do not generate `wasm` and `state` file if we don't need it (e.g raw chain-spec)

### CODE REVIEW

File: javascript/packages/orchestrator/src/paras.ts [hunk 1]
```diff
@@ -20,8 +20,9 @@ export async function generateParachainFiles(
   namespace: string,
   tmpDir: string,
   parachainFilesPath: string,
-  chainName: string,
+  chainName: string, //relay chain name
```
Comment (wirednkod) [inline javascript/packages/orchestrator/src/paras.ts:23]: I think we should rename the variable to `relayChainName` in order to be more clear

**Review by wirednkod — APPROVED** [review]
Some nip tucks - other than that looks good

File: javascript/packages/orchestrator/src/configGenerator.ts [hunk 2]
```diff
@@ -691,7 +691,7 @@ const prometheusExternal = (networkSpec: ComputedNetwork): boolean => {
     : true;
 };
 
-const getFirstCollatorCommand = (parachain: ParachainConfig): string => {
+export function getFirstCollatorCommand(parachain: ParachainConfig): string {
```
[the remark this answers is not in the document]
Comment (pepoviola) [reply javascript/packages/orchestrator/src/configGenerator.ts:694]: `export const function` is not a valid syntax ( const fn ... is valid in rust :))

File: javascript/packages/cli/src/cli.ts [hunk 3]
```diff
@@ -414,7 +414,7 @@ async function spawn(
   const dir = opts.dir || "";
   const force = opts.force || false;
   const monitor = opts.monitor || false;
-  const spawnConcurrency = opts.spawnConcurrency || 1;
+  const spawnConcurrency = opts.spawnConcurrency || 4;
```
[the remark this answers is not in the document]
Comment (pepoviola) [reply javascript/packages/cli/src/cli.ts:417]: Oks, the idea here is that in gral. spawning in batches of `4` would be `fine` in all the providers without create a bottleneck.

**Pull request merged**